phpcs.xml

```diff
@@ -1,5 +1,10 @@
 <?xml version="1.0"?>
 <ruleset name="DataValuesGeo">
+
+	<arg name="extensions" value="php"/>
+	<arg value="s"/>
+	<arg value="p"/>
```

Comment:
Setting the `-p` and `-s` flags like this is not common practice in MediaWiki projects.

Reply:
Not a MediaWiki project

Motivation TL;DR: there is a config file to put the config into

Reply:
IIRC setting -sp like this breaks integration with different IDEs, which is why we moved it out of phpcs.xml and into composer.json.

Reply:
Could you provide a link or some such? Cause if this is the case we might want to change how we do this in the FUN team.

Reply:
Try running `./vendor/bin/phpcs --report=json`, it should output a JSON object with all of the PHPCS issues, but now that -sp is hardcoded in, that comand also outputs the dots and time info, meaning that the output isn't valid JSON anymore.

I don't remember what the original bug report was anymore unfortunately, it was a few years ago.

Reply:
Ok, good point. Thanks for much for informing me. Will submit a PR for this later.

Reply:
https://github.com/DataValues/Geo/pull/124